### ISSUE TITLE
Inconsistent results between MacOS and Linux

### ISSUE THREAD

**Issue by ewconnell** [apple/swift-numerics#102]
Hi, my framework uses the Numerics package. We developed a model using the SwiftRT API and a version using TensorFlow. On the Mac we get the exact same results. The results were recorded and put into the unit tests to assure a constant point of reference.

On Ubuntu and gLinux, TensorFlow produces the exact same results. SwiftRT using the Numerics package almost produces the same results up to 0.001 difference. I am wondering if the Numerics package is relying on system library functions which are inconsistent across platforms?

If so, this is a big problem. Any insights?

Thanks, Ed

**Comment by SusanDoggie** [comment]
Is there example reproduce the problem?

**Comment by stephentyrone** [comment]
Yes, the system math library functions do not generally produce identical results on different platforms. It's a possible long-term goal for Swift Numerics to provide portable implementations, but it's *very* long-term. If you need exactly reproducible math library results in the short-term, you will need to link against a math library that you control for that specific purpose.
